pipelines/A4/queries.sql

```diff
@@ -0,0 +1,5 @@
+-- Wieviele Einträge enthält die Tabelle?
```

Comment:
The question is asked to the whole database, isn't it?
If so, here we would not know what table you are talking about

Reply:
I have done a better query now.